util/dao/utils/src/lib.rs

```diff
@@ -22,11 +23,19 @@ pub const DAO_VERSION: u8 = 1;
 pub const DAO_SIZE: usize = 32;
 
 pub fn genesis_dao_data(txs: Vec<&TransactionView>) -> Result<Byte32, Error> {
+    genesis_dao_data_with_satoshi_gift(txs, &H256([0u8; 32]), Ratio(1, 1))
+}
```

Comment:
Should tell people it's only for test purpose.

Reply:
IMO it's ok for this, logically the satoshi gift is not neccesary for the DAO, maybe in the future we need a testnet do not have a satoshi gift cell.

Since we have integration test , it's hard to misuse this method.

Reply:
I really hate it: `H160([0u8; 20])`. No body like `Shotgun Surgery`! Please call APIs!

**Avoid using constructor methods for those structs that has a lot of features!**